sdk/cosmos/microsoft-azure-cosmos/src/main/java/com/azure/data/cosmos/internal/AsyncDocumentClient.java

```diff
@@ -242,6 +240,14 @@ public TokenResolver getTokenResolver() {
         public void setTokenResolver(TokenResolver tokenResolver) {
             this.tokenResolver = tokenResolver;
         }
+
+        public CosmosKeyCredential getCosmosKeyCredential() {
```

Comment:
'get' should be removed.
@srnagar

Reply:
There are other getters in the AsyncDocumentClient, I believe either we should remove `get` from all of them, or from none. I am open to removing `get` from all of them.

Reply:
I would go for removing 'get'  and use no prefix., since we are using 'with' to set the values.
@srnagar , can you confirm this?

Reply:
As this is an internal type using the old get/set model, I don't think we need to block on changing this in this PR. We do need to change it all up for our internals before November.

Reply:
I'd prefer not to bloat the scope of the PR.

Reply:
the getter/setter in AsyncDocumentClient are not public methods.
IMO changing them not related to the scope of this PR.